.gitignore

```diff
@@ -8,6 +8,10 @@
 *.war
 *.ear
 
+# web building artifacts
+/website/.sass-cache
+/website/_site
```

Comment:
Let's remove the _site from .gitignore. Otherwise git status will not show the generated content as not belonging to the repo.